pkg/xds/envoy/clusters/common_lb_configurer.go

```diff
@@ -0,0 +1,25 @@
+package clusters
+
+import (
+	envoy_api "github.com/envoyproxy/go-control-plane/envoy/api/v2"
+)
+
+func CommonLb() ClusterBuilderOpt {
+	return ClusterBuilderOptFunc(func(config *ClusterBuilderConfig) {
+		config.Add(&CommonLbConfigurer{})
+		config.Add(&altStatNameConfigurer{})
+		config.Add(&timeoutConfigurer{})
```

Comment:
alt stat + timeout are only for Cluster "creator", leave here only commonLbConfigurer